src/Common/Constants.ts

```diff
@@ -351,6 +351,7 @@ export class HttpStatusCodes {
   public static readonly Created: number = 201;
   public static readonly Accepted: number = 202;
   public static readonly NoContent: number = 204;
+  public static readonly NotModified: number = 304;

```

Comment:
It feels like we should use some package for these definitions. Like [this](https://www.npmjs.com/package/http-status-codes)...

Reply:
Yep, that'd definitely help given the comprehensive list. I'd do a separate pass just for that cause it looks like we're using these Constants everywhere.